enterprise/frontend/src/embedding-sdk/hooks/private/use-init-data.ts

```diff
@@ -57,17 +57,8 @@ export const useInitData = ({ config }: InitDataLoaderParameters) => {
 
     api.basename = config.metabaseInstanceUrl;
 
-    const authErrorMessage = getAuthConfiguration(config, dispatch, appName);
-
-    if (authErrorMessage) {
-      dispatch(
-        setLoginStatus({
-          status: "error",
-          error: new Error(authErrorMessage),
-        }),
-      );
-    }
-  }, [appName, config, dispatch, loginStatus.status]);
+    getAuthConfiguration(config, dispatch);
```

Comment:
We should rename this function then. It's not returning anything, and actually has side effects that configure the configuration, so the previous name was also a bit misleading.

```suggestion
    configureAuthConfiguration(config, dispatch);
```

Reply:
Fixed in https://github.com/metabase/metabase/pull/47821/commits/60fba7f93be0c07cf5a1f1823abc41d15a0dc0c5, thanks!